react/src/lib/components/DeckGLMap/DeckGLMap.tsx

```diff
@@ -33,6 +33,7 @@ export interface DeckGLMapProps {
         visible?: boolean | null;
         cssStyle?: Record<string, unknown> | null;
         horizontal?: boolean | null;
+        reverseRange?: boolean | null;
```

Comment:
Not sure we need this as a global prop. I think it's enough that we have it as an option on the color legend itself.